data/soapvalidator/MailClient/HAB/ZCS-5705_ZimbraHABOu.xml

```diff
@@ -0,0 +1,341 @@
+			<t:response>
+				<t:select path="//admin:PingResponse" />
+			</t:response>
+		</t:test>
+
+	</t:test_case>
+
+	<t:test_case testcaseid="ZimbraHABCreateOu" type="always">
+		<t:objective>Account Setup </t:objective>
+		<t:steps>
+			1. Login into admin.
+			2. Create test accounts.
+		</t:steps>
+
+		<t:test id="admin_login" required="true">
+			<t:request>
+				<AuthRequest xmlns="urn:zimbraAdmin">
+					<name>${admin.user}</name>
+					<password>${admin.password}</password>
+				</AuthRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//admin:AuthResponse/admin:authToken" set="authToken" />
+			</t:response>
+		</t:test>
+
+		<t:test required="true">
+			<t:request>
+				<CreateDomainRequest xmlns="urn:zimbraAdmin">
+					<name>${domain.name}</name>
+					<a n="zimbraNotes">test of adding an OU</a>
+				</CreateDomainRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//admin:CreateDomainResponse/admin:domain"
+					attr="id" set="domain.id" />
+			</t:response>
+		</t:test>
+
+		<t:test required="true">
+			<t:request>
+				<CreateAccountRequest xmlns="urn:zimbraAdmin">
+					<name>${account1.name}</name>
+					<password>${defaultpassword.value}</password>
+					<a n="zimbraIsAdminAccount">TRUE</a>
+				</CreateAccountRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//admin:CreateAccountResponse/admin:account"
+					attr="id" set="account1.id" />
+				<t:select
+					path='//admin:CreateAccountResponse/admin:account/admin:a[@n="zimbraMailHost"]'
+					set="account1.server" />
+			</t:response>
+		</t:test>
+
+	</t:test_case>
+
+	<t:test_case testcaseid="ZimbraHABCreateOu_1" type="smoke"
+		bugids="ZCS-5705">
+		<t:objective>Create a zimbra organisational unit for a domain
+		</t:objective>
+		<t:steps>
+			1. Create a new HAB OU and assign to a domain
+			2. Verify the OU
+			gets created
+		</t:steps>
+
+
+		<t:test>
+			<t:request>
+				<HABOrgUnitRequest op="create" name="${ou1.name}"
+					xmlns="urn:zimbraAdmin">
+					<domain by="name">${domain.name}</domain>
+				</HABOrgUnitRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//admin:HABOrgUnitResponse//admin:habOrgUnitName"
+					match="${ou1.name}" />
+			</t:response>
+		</t:test>
+
+	</t:test_case>
+
+	<t:test_case testcaseid="ZimbraHABCreateOu_2" type="smoke"
+		bugids="ZCS-5705">
+		<t:objective>Rename an zimbra organisational unit for a domain
+		</t:objective>
+		<t:steps>
+			1. Rename an existing HAB OU assigned to a domain to a
+			newname
+			2. Verify
+			the OU gets renamed
+		</t:steps>
+
+		<t:test>
+			<t:request>
+				<HABOrgUnitRequest op="rename" name="${ou1.name}"
+					newName="${ou1.newname}" xmlns="urn:zimbraAdmin">
+					<domain by="name">${domain.name}</domain>
+				</HABOrgUnitRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//admin:HABOrgUnitResponse//admin:habOrgUnitName"
+					match="${ou1.newname}" />
+				<t:select path="//admin:HABOrgUnitResponse//admin:habOrgUnitName"
```

Comment:
Also verify the name is created

Reply:
WE don't have a getRequest to get the OU created or not. 1. In the response we can check the OU is present.
2. Try creating OU with same name and check it gives error as already exists which is covered as part of test case 5